syft/workers/plan.py

```diff
@@ -0,0 +1,261 @@
+from .base import BaseWorker
+from syft.codes import MSGTYPE
+import syft as sy
+import random
+
+
+class PlanPointer:
+    def __init__(self, id, location, id_at_location, owner):
+        self.id = id
+        self.location = location
+        self.id_at_location = id_at_location
+        self.owner = owner
+
+
+def replace_ids(obj, change_id, to_id, from_worker, to_worker):
```

Comment:
Should this just be a private class method? Is it ever called from outside this file

Reply:
Right!